src/Arcus.Templates.WebApi/.template.config/template.json

```diff
@@ -87,6 +95,10 @@
     "CertificateAuth": {
       "type": "computed",
       "value": "(authentication == \"Certificate\")"
+    },
+    "AppSettings": {
+      "type": "computed",
+      "value": "(authentication == \"Certificate\")"
```

Comment:
I would make this a seperate option for the template as this can be useful beyond this scope

Reply:
You mean that users are allowed to set if the project should include `appsettings.json` ?

Reply:
I added this because it would make more sense in the source files to use this directive than `CertificateAuth`. But in any case, beyond this scope indeed if you talk about having a seperate option.

Now it's just a compjuted one.

Reply:
It's certainly a good addition but would make it a general one since we don't include it by default, but folks can opt in for it if the want.

Maybe something like `includeAppSettings` or `useAppSettings`?

Reply:
Ok, but maybe in other PR; this PR is only about changing the certificate authentication information to the configuration file.